### PR TITLE
About care for carers

### PR DESCRIPTION
**Pull request by LizG11**
Adding in the page and place holder texr

### CODE REVIEW

**Review by AbbieC20 — APPROVED** [review]
Remove changes to gallery.svelte?

File: src/routes/about-care-for-carers.svelte [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<script>
+	import Container from '$lib/components/container.svelte';
+</script>
+
+<Container>
+    <h1>About Care for Carers</h1>
+    <br>
+    <div>
+    <h2>The Space in Deal</h2>
+<br>
```
Comment (lhollyer) [inline src/routes/about-care-for-carers.svelte:10]: Looking at the website these line breaks haven't left much of a gap between paragraphs. For consistency sake we should split each paragraph into its own `<p>` tag, like on the Contact Us page

File: src/routes/index.svelte [hunk 2]
```diff
@@ -6,16 +6,21 @@
...
-			labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
-			laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
-			voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
-			non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-			<br /> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
-			ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-			nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit
-			esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt
-			in culpa qui officia deserunt mollit anim id est laborum.
+			Pete Dennis is the founder of the SMALL society, the umbrella organisation for
```
Comment (lhollyer) [inline src/routes/index.svelte:9]: @LizG11 do we want to refer to TsS as `The small Society` or `The SMALL Society`?

Comment (LizG11) [reply src/routes/index.svelte:9]: I’ll check with Pete and confirm

Comment (lhollyer) [reply src/routes/index.svelte:9]: Are we happy to address this within the catchall branch https://github.com/lhollyer/the-small-society/tree/pre-launch-tweaks?

Comment (LizG11) [reply src/routes/index.svelte:9]: The website should read The small Society or abbreviation to TsS. Thanks for checking!

Comment (LizG11) [reply src/routes/index.svelte:9]: Happy to address in that branch

**Pull request merged**